Get moves by playbook

GonzaH opened the pull request:
I started this branch with an idea but I added some things so the name isn't as descriptive as I would wish. What I did was:

- Add a way to get all the movements by playbook, basic or adult (in the future I'll add special one like end of session and when you enter a fight as a team)
- Added a new field in the moves json so the move has info of whether it requires a dice roll or not
- Added all missing moves that don't require dice rolling
- Implemented a functionality that, if the move requires a roll, rolls the dice and adds it to the embed
- Created a way to get all basic playbooks as a list, also as a list containing 'The PlaybookName' so it looks more as in the playbook

harkano commented:
Looks like !help isn't working, or !moves+.  Are they being grabbed by get_moves()?

Is the syntax "!moves bull" supposed to print all basic and bull moves?

Also did you engage the modifier cap?  Doesn't look like it's applying it.

GonzaH commented:
> Looks like !help isn't working, or !moves+. Are they being grabbed by get_moves()?
> 
> Is the syntax "!moves bull" supposed to print all basic and bull moves?
> 
> Also did you engage the modifier cap? Doesn't look like it's applying it.

Crap, I don't know about the !moves+ or !help, I'll look into it in about 5 hours.

The syntax for the playbooks moves is !moves-bull and only returns the bull ones, for the basics is !moves-basic and for adults is !moves-adult.

About the cap it was the first thing I added, it was working before, perhaps when I moved the rolling logic it was broken, perhaps is for the best if you revert the merge until I have these bugs fixed.
